FirebaseInAppMessaging/Swift/Source/SwiftUIPreviewHelpers.swift

```diff
@@ -14,25 +14,25 @@
 
 import FirebaseInAppMessaging
 
-@available(iOS 13, tvOS 13, *)
+@available(iOS 13.0, macOS 10.15, macCatalyst 13.0, tvOS 13.0, *)
```

Comment:
Nit: FIAM isn't available on macOS or macCatalyst yet as far as I'm aware. This could be marked as unavailable instead.